CPAC/resources/configs/pipeline_config_abcd-prep.yml

```diff
@@ -46,6 +46,7 @@ anatomical_preproc:
     T1w_ACPC_template: /opt/dcan-tools/pipeline/global/templates/MNI152_T1_1mm.nii.gz
 
   brain_extraction:
+    run: On
```

Comment:
Is this change intentional?

Reply:
Yes, but I think I should leave this untouched. This config inherits from `blank` and `brain extraction` is `off` there by default. I will revert it back.

Reply:
Resolved in 58622451dcc56c1d3e74bbf98f46bfc74dc40da6